drivers/serial/uart_nrfx_uarte.c

```diff
@@ -62,7 +69,9 @@ struct uarte_async_cb {
 	 * transmission by uart_poll_out
 	 */
 	const uint8_t *volatile tx_buf;
```

Comment:
This no longer needs to be `volatile`. Also the comment above became outdated.

Reply:
done